Utilities/WDLs/PRSQC.wdl

```diff
@@ -95,34 +65,27 @@ task CheckScores {
     command <<<
         set -euo pipefail
 
-        cat <<'EOF' > check_scores_against_expected_values.py
+        cat <<'EOF' > check_scores.py
         import pandas as pd
 
-        scores = pd.read_csv('~{scores}', sep = '\t', header = 0)
+        prs_full_risk = pd.read_csv('~{prs_full_risk}', sep = '\t', header = 0)
         acceptable_range = pd.read_csv('~{acceptable_range}', sep = '\t', header = 0)
 
-        # Check whether PRS_SCORE and FULL_MODEL_SCORE are within the acceptable range
-        prs_score_passed = scores.loc[0, "PRS_SCORE"] >= acceptable_range.loc[0, "MIN"] and scores.loc[0, "PRS_SCORE"] <= acceptable_range.loc[0, "MAX"]
-        full_model_score_passed = scores.loc[0, "FULL_MODEL_SCORE"] >= acceptable_range.loc[1, "MIN"] and scores.loc[0, "FULL_MODEL_SCORE"] <= acceptable_range.loc[1, "MAX"]
+        # Check whether prs_score and combined_risk_score are within the acceptable range
+        prs_score_passed = prs_full_risk.loc[0, "prs_score"] >= acceptable_range.loc[0, "min"] and prs_full_risk.loc[0, "prs_score"] <= acceptable_range.loc[0, "max"]
```

Comment:
Instead of using the row indices, can we maybe have a column that says which value the min/max values correspond to? I'm thinking of a format like
```
metric    min    max
prs_score 10     20
combined_risk_score 11     21
pc1       -20    20
pc2       -20    20
```
That's easier to understand, harder to mess up, and is in line with the qc_thresholds file for imputation QC: `gs://broad-dsde-methods-bge-resources/GlimpseImputation/QCThresholds/1000G_HGDP_no_singletons/1000G_HGDP_no_singletons.qc_thresholds.tsv`